exercises/chapter4/test/Main.purs

```diff
@@ -6,6 +6,25 @@ import Data.Maybe (Maybe(..))
 import Data.Path (filename, root)
 import Data.Tuple (fst)
 import Effect (Effect)
+import Test.Solutions
+  ( allTrue
```

Comment:
It doesn't seem like this is rebased correctly on the latest solutions (or master) branch, since there shouldn't be any changes here outside of ch7

Reply:
Yes, I've noticed (and wondered about) that as well.  It's become tricky.